### DISCUSSION TITLE
How to use Google Maps (Satellite view) as a basemap?

### DISCUSSION THREAD

**Discussion by koberoi1** [developmentseed/lonboard#305] [Q&A]
Hi! 
I haven’t found a documentation to use Google maps as a basemap. Currently the documentation is restricted to DarkMatter, Positron & Voyager. Help is much appreciated. Thanks.

**Comment by kylebarron** [comment]
Using Google Maps as a basemap requires changing to use the Google Maps JavaScript API, which is not free to use. Users would have to register with Google Maps, get an API key, and pay for the maps they create with lonboard. It would also have some technical implications by adding another map renderer. It's technically possible to implement but unlikely without funding.

Your options are:

- Use raster tiles as a basemap through the `BitmapTileTayer` (not yet released but I'll make an 0.6 release soon)
- Use a Maplibre-compatible basemap style, such as but not limited to the ones described in the basemap section of the docs https://developmentseed.org/lonboard/latest/api/basemap/
